### ISSUE TITLE
[Enhancement] On-off switch colour 

### ISSUE THREAD

**Issue by xIUPITERx** [Rem0o/FanControl.Releases#1480]
Use a different colour for the on-off switch for a better contrast.

![image](https://user-images.githubusercontent.com/97352673/213865852-1c6844f3-806e-4095-a049-6d661d7ff5cf.png)

**Comment by Rem0o** [comment]
You can edit that color yourself. 
![image](https://user-images.githubusercontent.com/14118956/213879039-f3b00557-3e15-464b-92be-2e20a86cc08e.png)

**Comment by xIUPITERx** [comment]
Oh thanks, I completely missed that, but it's still not ideal.
If I would use a brighter colour, the background of the switch is very hard to see.
Maybe add a third colour or compare the switch-background with the primary colour to make sure the contrast is good.
![image](https://user-images.githubusercontent.com/97352673/213889602-dc385a58-660f-4d2b-889f-6b1d74a5b01f.png)
_The compression of the screenshot makes it even worse._

**Comment by xIUPITERx** [comment]
Could there be an option that makes all colours dependent on the Windows accent colour and then automatically selects a matching primary (the accent colour itself), secondary (a lighter variant) and a tertiary colour (darker than the accent)?

**Comment by Rem0o** [comment]
> Could there be an option that makes all colours dependent on the Windows accent colour and then automatically selects a matching primary (the accent colour itself), secondary (a lighter variant) and a tertiary colour (darker than the accent)?

Yes that's possible, but that won't come cheap, and would require quite a bit of work to make it nice.

Your specific situation is kinda meh because you are doing blue on blue. You would normally want your accent color to be of a contrasting color, like here a green/yellow, even white would work.

**Comment by xIUPITERx** [comment]
> but that won't come cheap,

What do you mean by that?

>You would normally want your accent color to be of a contrasting color

Yeah that's a problem, but you wont find an app on my pc which is not blue ;)
